Remind_Device/Remind_Device.ino

```diff
@@ -17,29 +17,32 @@ LiquidCrystal_I2C lcd(0x27, 2, 1, 0, 4, 5, 6, 7, 3, POSITIVE);
 
 // sets constant state variables from 1 to 7
 const int STATE_GENERAL_WAIT = 1;
-
 const int STATE_LCD_ON = 2;
-
 const int STATE_FILE_WAIT = 3;
-
 const int STATE_AUDIO_SELECT = 4;
-
 const int STATE_SPEAKER_PLAY = 5;
-
 const int STATE_ACKNOWLEDGE_WAIT = 6;
-
 const int STATE_MOTION_OFF = 7;
 
-// setting other variables
-int IndicatorLED = 8;
-int ConfirmationButton = 9;
-int ConfirmationButtonState = 1;
 
-// TO DO: button state variable, joystick state variable, motion sensor state variable, speaker count variable, and whatever WAVE needs
+// declaring digital pins
+int IndicatorLED = 8;            // Indicator LED output
+int ConfirmationButton = 9;      //acknowledge button? Input
+int MotionSensor = 10;           // Motion sensor input
+
+// declaring analog pins
+int XJoystick = A0;              // x-axis of joystick
+int YJoystick = A1;              // y-axis of joystick
+int JoystickButton = A2;         // joystick button treat as digital
+
+// declare variable data holders
+int SpeakerCount;                // For WAVE shield  -  not sure what it does
+int ConfirmationButtonState;     // Do we need button state? could it just be in an if statement?
```

Comment:
I'm not sure on this one. We might be able to? If you think it can be done with just an if statement with not button state variable, go ahead and give it a try. I can test it once you got an idea